src/views/List.jsx

```diff
@@ -38,21 +27,7 @@ export function List({ data, listToken }) {
 				</div>
 			) : (
 				<div>
-					<form onSubmit={handleSubmit}>
-						<label htmlFor="searchTerm">Filter list items</label>
-						<input
-							type="text"
-							value={searchTerm}
-							id="searchTerm"
-							placeholder="Start typing here..."
-							onChange={onSearch}
-						/>
-						{!!searchTerm && (
-							<button onClick={onSearchReset} type="reset">
-								Reset
-							</button>
-						)}
-					</form>
+					<SearchBar searchTerm={searchTerm} setSearchTerm={setSearchTerm} />
```

Comment:
Love that you created a new component! It's cleaner, and it makes good use of compartmentalizing the search feature.